src/classifier/Features.h

```diff
@@ -50,4 +55,76 @@ class Features {
   Pipe *pipe_; // The pipe that owns this feature handler.
 };
 
+
+//struct  FeatureLabelPair : Structure to define a a feature-label pair
+struct FeatureLabelPair {
+	uint64_t feature;
+	int label;
+};
+
+//struct  FeatureLabelPairMapper: Structure to define a hash function and a comparison function for FeatureLabelPair
+struct FeatureLabelPairMapper {
```

Comment:
I don't think we need this class. Features are automatically conjoined with labels via the SparseLabeledParameter class. But I may be missing something, so we can discuss this further if necessary.